src/components/GenerateGifForm.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import './GenerateGifForm.css';
+
+class GenerateGifForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { name: '', titleText: '' };
+    this.handleInputUpdate = this.handleInputUpdate.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(evt) {
+    evt.preventDefault();
+    this.props.handleGenerateGIF();
+    this.props.updateGifFileName(this.state.name);
+    // set title
+    this.setState({ name: '', titleText: '' });
```

Comment:
i wouldn't worry about clearing the form on submit -- if the user adds more frames after the initial download, we can pre-populate the old form values from redux. But if they clear out the frames, we should also clear out this form state from redux.

Reply:
Done